components/ReposTableComponents/SortAndFilterDropdowns.tsx

```diff
@@ -32,53 +32,38 @@ export const theme = createTheme({
 	},
 });
 
-export type SortSettings = {
-	type: "name" | "rank" | "time" | "internal" | "external" | "total" | "users";
-	direction: boolean;
-};
+export type SortSettings = { type: "name" | "repo" | "rank" | "time" | "internal" | "external" | "total" | "users", direction: boolean }
 
 export function SortBox(sortSetting: SortSettings, handleSortChange: any) {
-	return (
-		<ThemeProvider theme={theme}>
-			<FormControl className={styles.sortby} fullWidth>
-				<InputLabel
-					id="sort-by-select-label"
-					sx={{ fontSize: "1.3em", transform: "translate(10px, -15px)" }}
-				>
-					Sort by
-				</InputLabel>
-				<Select
-					label="_____ Sort by" //DO NOT REMOVE UNDERSCORES, label is only used for layout, see here https://mui.com/material-ui/api/outlined-input/#props
-					labelId="sort-by-select-label"
-					value={sortSetting.type}
-					onChange={handleSortChange}
-					IconComponent={(props) => {
-						console.log("Icon props:", props);
-						return (
-							<ArrowDropDownIcon
-								{...props}
-								fontSize="large"
-								htmlColor="#000000"
-							/>
-						);
-					}}
-				>
-					<MenuItem value="">
-						{" "}
-						<em>None</em>{" "}
-					</MenuItem>
-					<MenuItem value={"name"}>Name</MenuItem>
-					<MenuItem value={"rank"}>Rank</MenuItem>
-					<MenuItem value={"time"}>Time</MenuItem>
-					<MenuItem value={"internal"}>Internal count</MenuItem>
-					<MenuItem value={"external"}>External count</MenuItem>
-					<MenuItem value={"total"}>Total count</MenuItem>
-					<MenuItem value={"users"}>User count</MenuItem>
-					<MenuItem value={"mostOutdated"}>Most Outdated Dependency</MenuItem>
-				</Select>
-			</FormControl>
-		</ThemeProvider>
-	);
+
+	return <ThemeProvider theme={theme}>
+		<FormControl className={styles.sortby} fullWidth>
+			<InputLabel id="sort-by-select-label" sx={{ fontSize: '1.3em', transform: 'translate(10px, -15px)' }}>
+				Sort by
+			</InputLabel>
+			<Select
+				label="_____ Sort by" //DO NOT REMOVE UNDERSCORES, label is only used for layout, see here https://mui.com/material-ui/api/outlined-input/#props
+				labelId="sort-by-select-label"
+				value={sortSetting.type}
+				onChange={handleSortChange}
+				IconComponent={(props) => {
+					console.log("Icon props:", props)
+					return <ArrowDropDownIcon {...props} fontSize='large' htmlColor="#000000" />
+				}}
+			>
+				<MenuItem value=""> <em>None</em> </MenuItem>
+				<MenuItem value={"name"}>Name</MenuItem>
+				<MenuItem value={"repo"}>Repository</MenuItem>
+				<MenuItem value={"rank"}>Rank</MenuItem>
+				<MenuItem value={"time"}>Time</MenuItem>
+				<MenuItem value={"internal"}>Internal count</MenuItem>
+				<MenuItem value={"external"}>External count</MenuItem>
+				<MenuItem value={"total"}>Total count</MenuItem>
+				<MenuItem value={"users"}>User count</MenuItem>
+				<MenuItem value={"mostOutdated"}>Most Outdated Dependency</MenuItem>
```

Comment:
Just a small thing but can we maybe get the values in the drop down boxes to have the same fonts as the rest of the app? Maybe we need to add it as part of the theme for MuiMenuItem.
![Screen Shot 2022-10-11 at 8 41 40 am](https://user-images.githubusercontent.com/81656054/194956656-b62ca4be-1bc2-4d59-9626-6f174e61b7b9.png)

Reply:
Done in recent push.